<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UtQtGradientLegendUi</class>
 <widget class="QWidget" name="UtQtGradientLegendUi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>112</width>
    <height>268</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QWidget" name="gradientGrid" native="true">
     <layout class="QGridLayout" name="gridLayout_4" columnstretch="1,0">
      <item row="19" column="1" rowspan="2">
       <widget class="QLabel" name="tickLabel2">
        <property name="text">
         <string>0.1</string>
        </property>
       </widget>
      </item>
      <item row="13" column="1" rowspan="2">
       <widget class="QLabel" name="tickLabel5">
        <property name="text">
         <string>0.4</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1" rowspan="2">
       <widget class="QLabel" name="tickLabel9">
        <property name="text">
         <string>0.8</string>
        </property>
       </widget>
      </item>
      <item row="11" column="1" rowspan="2">
       <widget class="QLabel" name="tickLabel6">
        <property name="text">
         <string>0.5</string>
        </property>
       </widget>
      </item>
      <item row="15" column="1" rowspan="2">
       <widget class="QLabel" name="tickLabel4">
        <property name="text">
         <string>0.3</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" rowspan="2">
       <widget class="QLabel" name="tickLabel11">
        <property name="text">
         <string>1.0</string>
        </property>
       </widget>
      </item>
      <item row="9" column="1" rowspan="2">
       <widget class="QLabel" name="tickLabel7">
        <property name="text">
         <string>0.6</string>
        </property>
       </widget>
      </item>
      <item row="7" column="1" rowspan="2">
       <widget class="QLabel" name="tickLabel8">
        <property name="text">
         <string>0.7</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1" rowspan="2">
       <widget class="QLabel" name="tickLabel10">
        <property name="text">
         <string>0.9</string>
        </property>
       </widget>
      </item>
      <item row="21" column="1" rowspan="2">
       <widget class="QLabel" name="tickLabel1">
        <property name="text">
         <string>0.0</string>
        </property>
       </widget>
      </item>
      <item row="17" column="1" rowspan="2">
       <widget class="QLabel" name="tickLabel3">
        <property name="text">
         <string>0.2</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" rowspan="20">
       <widget class="UtQtGradientBar" name="gradientBar" native="true">
        <property name="minimumSize">
         <size>
          <width>20</width>
          <height>0</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLabel" name="variableLabel">
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Color</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>UtQtGradientBar</class>
   <extends>QWidget</extends>
   <header>UtQtGradientBar.hpp</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
